CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Next release
 
+- fix: Ensure transaction checks are compatible with starknet-rs
```

Comment:
I moved this one from the previous release because the change didn't make it into that release according to the release notes.